app/javascript/components/miq-structured-list/miq-structured-list-body/value-tags/miq-structured-list-text.jsx

```diff
@@ -4,8 +4,12 @@ import classNames from 'classnames';
 
 /** Component to print the text value inside a cell. */
 const MiqStructuredListText = ({ value }) => {
-  const text = (value === null || value === undefined ? '' : String(value));
-
+  let text;
+  if (Array.isArray(value)) {
+    text = value.join('\n');
```

Comment:
Instead of `\n`, should we be using `<br>` ?  I'm kind of surprised that `\n` works.

Reply:
I tried using <br> but it didn't seem to work. I might be doing it wrong but when I do `text = value.join('<br>');` the <br> shows up as actual text instead of a line break